src/main/kotlin/com/theworkshopvlc/chupapuntes/categories/domain/Category.kt

```diff
@@ -1,16 +1,11 @@
 package com.theworkshopvlc.chupapuntes.categories.domain
 
-import javax.persistence.Entity
-import javax.persistence.GeneratedValue
-import javax.persistence.GenerationType
-import javax.persistence.Id
-import javax.persistence.Table
+import javax.persistence.*
```

Comment:
This will make Travis to fail because it doesn't like this format.

Reply:
Now Travis will be happy https://github.com/theworkshopvlc/Chupapuntes/commit/10ead93b8aaee421eab28feba339ad5ce326fb21